src/FINDOLOGIC/Exceptions/ServiceNotAliveException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace FINDOLOGIC\Exceptions;
+
+class ServiceNotAliveException extends \RuntimeException
+{
+    public function __construct($alivetestUrl)
+    {
+        parent::__construct(sprintf('The service is not alive! Alivetest URL was %s.', $alivetestUrl));
```

Comment:
The purpose of the library is to hide implementation details, so showing one (the URL) in an exception provides little value.